CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing
```

Comment:
It looks like this was copied from the `python-sdk`. Please make sure to update all the repo paths in the first. Also, make sure the full doc applies to the contrib repo.